src/components/modules/dragNDrop.ts

```diff
@@ -13,39 +13,82 @@ export default class DragNDrop extends Module {
    */
   private isStartedAtEditor = false;
 
+  /**
+   * Listener identifiers
+   */
+  private listenerIds: string[] = [];
```

Comment:
It seems we can move this to base `Module` class and add there `listeners` property as a wrapper for `Listeners` module methods as it doesn't use anything from `this.Editor`

```
class Module {
  private listenersIds: string[] = [];

  protected listeners = {
    on(...) {
      this.listenersId.push(Listeners.on(...))
    },
    off() {},
    clearAll() {},
    ...
  }
}
```

Reply:
it is a good idea. Maybe